Rules/CommonScripts/AdminInventoryView.as

```diff
@@ -0,0 +1,71 @@
+void onRender(CRules@ this)
+{
+	CMap@ map = getMap();
+	if (map is null) return;
+
+	CPlayer@ player = getLocalPlayer();
+	if (player is null) return;
+
+	CControls@ controls = player.getControls();
+	if (controls is null) return;
+
+	Vec2f mouse_pos = controls.getMouseWorldPos();
+
+	if (player.isMod() && player.getTeamNum() == this.getSpectatorTeamNum() && controls.isKeyPressed(KEY_LCONTROL))
```

Comment:
You should remove this script if player is not a mod, this would save having to all the script every frame.

OR

Remove this script from all gamemode.cfg and add onChangeTeam in core gamemode files. Then check to see if an admin has switched to spec (if so, add file, and remove when they leave). Then you can get rid of checking isMod and getTeamNum every frame.

Along with that, define it client only at top of the script